tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -36,9 +36,9 @@ Here is a complete class example based on these guidelines:
     signal state_changed(previous, new)
 
     export var initial_state = NodePath()
-    var is_active = true setget set_is_active
+    var is_active = true set_is_active
```

Comment:
The correct in-line syntax for this is:
`var is_active = true: set = set_is_active`
Just removing `setget` doesn't work.